packages/frontend-shared/src/locales/en-US.json

```
@@ -597,8 +597,8 @@
       "title": "Migration Helper",
       "description": "Complete the steps below to migrate your project to Cypress 10",
       "step1": {
-        "title": "Rename existing specs",
-        "description": "In this step, we'll automatically rename and move your existing spec files.",
+        "title": "Migrate existing specs",
+        "description": "In this step, we'll automatically rename and/or move your existing spec files as needed.",
```

Simpler version, up to you! 
```suggestion
        "description": "In this step, we'll automatically update your existing spec files.",
```

---

Hahaha just caught up on the whole thread on wording between you and @marktnoonan . My 2 cents is that `rename and/or move` is a bit wordy and makes me think, whereas "update" covers both and I don't have to cerebrally digest what's happening. It's a simple thing we're doing, imo the markup should be short & sweet

---

When I read "update" I feel like like it could do absolutely anything to my files. It's not clear that the changes are limited to moving/renaming, that in this step we aren't going to actually touch the internals of files (like we do other places).

---

What if it's "update your existing spec file paths" instead of "update your existing spec files"?

fwiw I'm not married to either approach and will support whatever you choose 😃

---

I'm happy with either, just gonna ship this as is. I think they are both fine and combined with the UI, it's pretty clear what's going on.